resources/assets/icons/brands/ark_black.svg

```diff
@@ -1 +1 @@
-<svg version="1.1" id="Слой_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0" y="0" viewBox="0 0 1200 1201" xml:space="preserve"><defs><path id="SVGID_1_" d="M0 0h1200v1201H0z"/></defs><clipPath id="SVGID_2_"><use xlink:href="#SVGID_1_" overflow="visible"/></clipPath><g id="Logo_ARK.io_Black"><g id="logo" transform="translate(-655 -116)"><path id="Rectangle" d="M715 116h1080c33.1 0 60 26.9 60 60v1080c0 33.1-26.9 60-60 60H715c-33.1 0-60-26.9-60-60V176c0-33.1 26.9-60 60-60z" fill="currentColor"/><path id="ARK" d="M1253.4 608.2l-456.2 472.3 456.2-730 456.1 730.1-456.1-472.4zm-135.3 245.9H1388l77.9 80.6h-425.1l77.3-80.6zm56.8-58.3l78.5-81.3 79.4 81.4-157.9-.1z" fill="#212225"/></g></g></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 1200 1201"><defs><path id="SVGID_1_" d="M0 0h1200v1201H0z"/></defs><clipPath id="SVGID_2_"><use xlink:href="#SVGID_1_" overflow="visible"/></clipPath><g id="Logo_ARK.io_Black"><g id="logo" transform="translate(-655 -116)"><path id="Rectangle" d="M715 116h1080c33.1 0 60 26.9 60 60v1080c0 33.1-26.9 60-60 60H715c-33.1 0-60-26.9-60-60V176c0-33.1 26.9-60 60-60z" fill="currentColor"/><path id="ARK" d="M1253.4 608.2l-456.2 472.3 456.2-730 456.1 730.1-456.1-472.4zm-135.3 245.9H1388l77.9 80.6h-425.1l77.3-80.6zm56.8-58.3l78.5-81.3 79.4 81.4-157.9-.1z" fill="#212225"/></g></g></svg>
```

Comment:
I'm not a big fun of `xlink:href=""` it can cause strange error on page loading that can be difficult to guess, if the ID clash with another in the page.
In this case I'd prefer treat it manually by copy/pasting it on https://jakearchibald.github.io/svgomg/ and check the result. 90% of the time you end up with an identical icon with less markup. But again it's so boring...

For example, this is the optimized version from svgomg ⬇️ 
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1200 1201"><defs><path id="a" d="M0 0h1200v1201H0z"/></defs><path d="M60 0h1080c33.1 0 60 26.9 60 60v1080c0 33.1-26.9 60-60 60H60c-33.1 0-60-26.9-60-60V60C0 26.9 26.9 0 60 0z" fill="currentColor"/><path d="M598.4 492.2 142.2 964.5l456.2-730 456.1 730.1-456.1-472.4zM463.1 738.1H733l77.9 80.6H385.8l77.3-80.6zm56.8-58.3 78.5-81.3 79.4 81.4-157.9-.1z" fill="#212225"/></svg>
```